src/pages/game/[id].tsx

```diff
@@ -1,135 +1,94 @@
+  const addons = trpc.addon.all.useInfiniteQuery(
+    { limit: 7, game_id: game.id },
+    { getNextPageParam: (lastPage) => lastPage.nextCursor }
   );
 
-  if (scene === Scene.POPULATE) return (
-    <LayoutComponent game={game}>
-      <PopulateScene
-        players={players}
-        addPlayer={addPlayer}
-        removePlayer={removePlayer}
-        loadPlayers={loadPlayers}
-        setScene={setScene}
-      />
-    </LayoutComponent>
-  );
+  async function scrolling(e: UIEvent) {
+    const { scrollTop, scrollHeight, clientHeight } = e.currentTarget;
+    if ((scrollTop + clientHeight === scrollHeight) && !addons.isFetching) {
+      console.log("fetching...");
+      await addons.fetchNextPage();
+    }
+  }
 
-  if (scene === Scene.ONGOING) return (
-    <LayoutComponent game={game}>
-      <GameScene
-        players={players}
-        addPlayer={addPlayer}
-        removePlayer={removePlayer}
-        shufflePlayers={shufflePlayers}
-        settings={settings}
-        cards={cards}
-      />
+  return (
+    <LayoutComponent title={game.title}>
+      <main className={styles.main}>
+
+        {/* Search Section */}
+        <form role="search" className={styles.horizontalList}>
+          <ButtonComponent variant="Secondary" >official</ButtonComponent>
+          <ButtonComponent variant="Secondary" ><BsWifi /></ButtonComponent>
+          <ButtonComponent variant="Secondary" ><BsWifiOff /></ButtonComponent>
+          <input placeholder="E.g 'Base pack'" />
+        </form>
+
+        <hr />
+
+        {/* List of addons */}
+        <section className={styles.addons} onScroll={scrolling}>
+          {addons.data?.pages.map(page =>
+            page.items.map(x =>
+              <AddonComponent key={x.id} addon={x} />
+            )
+          )
+          }
+        </section>
+
+        {/* Card count and settings */}
+        <section className={styles.horizontalList}>
+          <TagComponent><BsWifi />125</TagComponent>
+          <TagComponent><BsWifiOff />125</TagComponent>
+          <ButtonComponent variant="Secondary">Settings</ButtonComponent>
+        </section>
+
+        {/* Start */}
+        <section className={styles.horizontalList}>
+          <ButtonComponent>Start Online</ButtonComponent>
+          <ButtonComponent>Start Offline</ButtonComponent>
+        </section>
+
+      </main>
     </LayoutComponent>
   );
-
-  return <div>Error</div>;
 };
 
-export default GamePage;
-
-
 interface Props {
-  game: Game;
-  addons: Addon[];
+  game: Game
 }
 
+export default GameSetup;
 
-export async function getStaticPaths() {
-  const games = await prisma.game.findMany();
-  const paths = games.map(game => ({ params: { id: game.id } }));
-
+export const getStaticPaths: GetStaticPaths = async () => {
+  const result = await prisma.game.findMany();
+  const paths = result.map((project) => ({ params: { id: project.id } }));
   return {
-    paths,
-    fallback: false,
+    paths: paths,
```

Comment:
This could be done with a shorthand as it was before
```suggestion
    paths,
```